crates/amaru/src/pipeline.rs

```diff
@@ -15,14 +15,14 @@ pub type DownstreamPort = gasket::messaging::OutputPort<BlockValidationResult>;
 
 pub struct Stage<S>
 where
-    S: Store,
+    S: Store + Send + Sync,
```

Comment:
The `Sync` bound is not needed: the Stage isn't accessed simultaneously from multiple threads.